app/modules/site_settings/helpers.py

```diff
@@ -342,6 +388,27 @@ def is_valid_value_for_class(cls, cls_name, cfd_id, value):
             return False
         return True
 
+    # DEX-1337 PATCH op=remove path=site.custom.customFields.CLASS/GUID must be supported
+    #   this also will be called for (currently unused) PATCH op=remove path=site.custom.customFieldCategories fwiw (see DEX-1362)
+    #   force_removal=True means go ahead and blow away all the data that exists.  ouch!
+    #
+    #   note: this *intentionally* disallows straight-up reseting entire customFields.CLASS object.  youre welcome.
+    @classmethod
+    def patch_remove(cls, key, force_removal=False):
+        import re
+
+        m = re.search(r'site.custom.customFields.(\w+)/([\w\-]+)', key)
+        assert m and len(m.groups()) == 2
+        class_name = m.group(1)
+        cf_id = m.group(2)
+        defn = cls.get_definition(class_name, cf_id)
+        if not defn:
+            raise ValueError(f'invalid guid {cf_id} for class {class_name}')
+        # FIXME _drop_data does not yet take into consideration force=True so this needs fixing
```

Comment:
Is there a PR associated with this FIXME? And didn't we standardise on TODO?

Reply:
DEX-1367 and i dont really remember.  depends on the meaning of "we" and "standardise" here.  :smile: